config/locals.json

```diff
@@ -3,7 +3,7 @@
     "title": "Etcher",
     "prefix": "",
     "slogan": "Burn. Better.",
-    "lead": "Burn images to SD cards & USB drives, safe & easy.",
+    "lead": "Burn images to SD cards & USB drives, safely and easily.",
```

Comment:
Yea, we actually don't need that file in the git history. It's just used as cache at build time, I'll rm it.